src/main/java/edu/ie3/datamodel/models/timeseries/repetitive/LoadProfileInput.java

```diff
@@ -42,12 +40,17 @@ public PValue calc(ZonedDateTime time) {
 
   @Override
   protected Optional<ZonedDateTime> getPreviousDateTime(ZonedDateTime time) {
-    return Optional.of(time.minus(1, HOURS));
+    return Optional.of(time.minusHours(1));
   }
 
   @Override
   protected Optional<ZonedDateTime> getNextDateTime(ZonedDateTime time) {
-    return Optional.of(time.plus(1, HOURS));
+    return Optional.of(time.plusHours(1));
+  }
+
+  @Override
+  public List<ZonedDateTime> getTimeKeysAfter(ZonedDateTime time) {
+    return List.of(time.plusHours(1));
```

Comment:
This does not seem correct. However, I'm not sure if we need an actual implementation here

Reply:
It is needed to implement this method because this class extends `TimeSeries`. The way I implemented this is in line with the implementations of the methods `getPreviousDateTime` and `getNextDateTime`.

Reply:
We agreed that we keep the dummy value for now on